packages/libs/src/sdk/examples/signedRequest/index.ts

```diff
@@ -0,0 +1,57 @@
+import { sdk } from '@audius/sdk'
+import prompt from 'prompt-sync'
+
+/**
+ * Creates and sdk instance and logs in with user auth
+ * Fetches the listening history for that user
+ * Please set:
+ *  <email@email.com>
+ *  <password>
+ *  <my-handle>
+ */
+const main = async () => {
+  // Create an sdk with an app name and user auth
+  const audiusSdk = sdk({
+    appName: 'Example',
+    userAuth: true
+  })
+
+  // Sign in as our user, catching the first attempt to add
+  // the one-time-pass code.
+  try {
+    await audiusSdk.services.auth.signIn?.({
```

Comment:
I don't think we should ever promote accessing `sdk.services` directly. Also don't love the new `userAuth: true` setting - I think we should stick to the injected services model.

```ts
import { sdk, HedgehogAuth } from '@audius/sdk'

const authService = new HedgehogAuth()

await authService.signIn(...) // should also be possible after initializing sdk iiuc

const audiusSdk = new sdk({ appName: 'example', services: { auth: authService } })
```

I don't think optional `signIn` etc methods belong on the contract interface for `AuthService`

Reply:
I really dislike this approach... Feels very not native to a user experiencing using this library for the first time.

Reply:
I understand that that maps to a lot of our existing patterns w/ this lib... but essentially we make the user inherit all of the complexity of how we've built the library, rather than providing them an interface that they can use directly

Reply:
I now am in favor and will make these changes! ty... chatted offline